src/libraries/System.Private.Xml/src/System/Xml/Core/XmlRawWriter.cs

```diff
@@ -198,7 +265,7 @@ public override void WriteValue(DateTimeOffset value)
         {
             // For compatibility with custom writers, XmlWriter writes DateTimeOffset as DateTime.
             // Our internal writers should use the DateTimeOffset-String conversion from XmlConvert.
-            WriteString(XmlConvert.ToString(value));
```

Comment:
This comment seems not relevant as even now it is using 
https://github.com/dotnet/runtime/blob/abb5c348f242e546b5768ad70ebe4051f08f7d24/src/libraries/System.Private.Xml/src/System/Xml/XmlConvert.cs#L746

Reply:
Maybe it is connected with base code ?
https://github.com/dotnet/runtime/blob/8f33fd5cac1133a187bb27602f940608b8080758/src/libraries/System.Private.Xml/src/System/Xml/Core/XmlWriter.cs#L275

Reply:
@krwq Do you know if somebody is changing this Mode ?
https://github.com/dotnet/runtime/blob/448680517a26984227ac1e8a18b90e9a5d43d37b/src/libraries/System.Private.Xml/src/System/Xml/Serialization/Xmlcustomformatter.cs#L20-L34

I saw similar code in XmlSerializers and the only way to change is with Reflection and directly manipulating fiedl.